.bazelci/presubmit.yml

```diff
@@ -108,6 +108,7 @@ tasks:
       - "--apple_crosstool_top=@local_config_apple_cc//:toolchain"
       - "--crosstool_top=@local_config_apple_cc//:toolchain"
       - "--host_crosstool_top=@local_config_apple_cc//:toolchain"
+      - "--local_test_jobs=2"
```

Comment:
Sorry I just check the job already runs with `--local_test_jobs=2`, which is added automatically by the CI script
https://buildkite.com/bazel/rules-go-golang/builds/6622#019096cb-8b58-4881-840b-42f5466ad948

I think you might have to actually bump the test sizes of those timeout tests.